cpu_map_keyme2560.h

```diff
@@ -49,9 +49,9 @@
 
 /* Chip select for SRAM */
 #define SCS_SRAM_PORT           PORTC
-#define SCS_SRAM_PIN            PC3
+#define SCS_SRAM_PIN            PC2
 #define SCS_SRAM_DDR            DDRC
-#define SCS_SRAM_DDR_PIN        DDC3
+#define SCS_SRAM_DDR_PIN        DDC2
```

Comment:
`git blame` says that \*someone\* made it PC3 instead of PC2, but according to the schematic, it should be PC2. We didn't have this driver on a previous version of the board, so nothing changed. We didn't use this pin before now and that is why Grbl worked perfectly with this bug. The changed line was originally added on 5/23 in https://github.com/keyme/grbl/pull/112.